specification/deviceregistry/DeviceRegistry.Management/tspconfig.yaml

```diff
@@ -17,6 +17,10 @@ options:
     namespace: "com.azure.resourcemanager.deviceregistry"
     service-name: "Device Registry"
     examples-directory: "examples"
+  "@azure-tools/typespec-csharp":
+    namespace: "Azure.ResourceManager.DeviceRegistry"
```

Comment:
Where do you specify it is arm?

Reply:
As explained offline, once we set the correct namespace, the arm flag has been set correctly somewhere already.

Reply:
And the arm flag is based on `@armProviderNamespace` decorator, not the namespace.